### ISSUE TITLE
Only found you now

### ISSUE THREAD

**Issue by rparree** [makotot/assemble-index-json#7]
We need a better assemble community! i only just found your solution after spending today on https://www.npmjs.com/package/assemble-json-index

**Comment by makotot** [comment]
Yeah, it is hard to find out the assemble plugin that is already exist.
Searchable list of assemble plugins like [postcss.parts](http://postcss.parts/) would be helpful.

With regard to assemble-index-json, the keywords in package.json might should include `index` and `json`.
